CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## StreamChat
 ### ✅ Added
 - `quotesEnabled` property is added to the `ChannelConfig` [#1891](https://github.com/GetStream/stream-chat-swift/issues/1891)
+- Add support for  multiple active channel lists at the same time [#1879](https://github.com/GetStream/stream-chat-swift/pull/1879)
```

Comment:
To me, it feels more like a fix because we didn't explicitly forbid customers to have multiple channel lists. But leaving it up to you 👍

Reply:
Well, I get your point. But effectively we never had support for it, so we are adding it 🤔

Reply:
Nope, we did. Using the delegates, you can have multiple active channel lists, it's also used in DemoApp

Reply:
Understood! Changed it